lib/hooks/typography/basekick.ts

```diff
@@ -1,52 +0,0 @@
-interface BaseKickOptions {
```

Comment:
Let's pour one out for Basekick. Farewell, old friend. 🍻👋

Reply:
🌅 🥃 🥃 😢 👋